[Issue 8130] Refine BookKeeper metadata service uri when init metadata (#8269)
apache/pulsar: site2/docs/deploy-bare-metal.md

> --broker-service-url-tls pulsar+ssl://host1:6651,host2:6651,host3:6651
> ```

> > If you want to use an existing BookKeeper cluster, you can add `--existing-bk-metadata-service-uri` flag at the mean time like below:

Reviewer: ```suggestion
> If you want to use an existing BookKeeper cluster, you can add the `--existing-bk-metadata-service-uri` flag as follows:
```

Author: done.

>
> ```properties
> --existing-bk-metadata-service-uri "zk+null://bk1:2181;bk2:2181/ledgers" \
> --web-service-url http://host1:8080,host2:8080,host3:8080 \
> --web-service-url-tls https://host1:8443,host2:8443,host3:8443 \
> --broker-service-url pulsar://host1:6650,host2:6650,host3:6650 \
> --broker-service-url-tls pulsar+ssl://host1:6651,host2:6651,host3:6651
> ```

> > The metadata service uri of existing BookKeeper cluster can be fetched via `bin/bookkeeper shell whatisinstanceid` command. Note that the value need to be enclosed in double quotes since the multi metadata service uri is semicolon separated.

Reviewer: ```suggestion
> You can obtain the metadata service URI of the existing BookKeeper cluster by using the `bin/bookkeeper shell whatisinstanceid` command. You must enclose the value in double quotes since the multiple metadata service URIs are separated with semicolons.
```

Author: thanks, done.
